src/test/java/lambda/part3/exercise/Mapping.java

```diff
@@ -50,6 +50,31 @@ public MapHelper(List<T> list) {
         }
     }
 
+    private List<JobHistoryEntry> addOneYear(List<JobHistoryEntry> list){
+        int size = list.size();
+        for(int i =0; i < size; i++){

```

Comment:
Why not `for each`?
Estimate complexity for `LinkedList`.
Use `MapHelper` here.